class UserAlreadyInOrganizationException(Exception):
    pass


class UserNotInOrganizationException(Exception):
    pass
